app/controllers/targets_controller.rb

```diff
@@ -15,10 +15,14 @@ def create
     @target = Target.new(target_params)
     @target.user_id = current_user.id
 
-    if @target.save
-      render json: @target, status: :created
+    if current_user.targets.count >= 3
```

Comment:
Having this here it works, but what about moving this validation to the model? This allows you to validate this condition in every part of the application, not only in the target index method 😉 

https://guides.rubyonrails.org/active_record_validations.html#custom-methods
https://weblog.jamisbuck.org/2006/10/18/skinny-controller-fat-model